cli/cli.go

```diff
@@ -68,6 +70,21 @@ func (c *Cli) NewAPIClient() {
 	c.APIClient = client
 }
 
+// InitLog initializes log Level and log format of client.
+func (c *Cli) InitLog() {
+	if c.Option.Debug {
+		logrus.SetLevel(logrus.DebugLevel)
+		logrus.Infof("start client at debug level")
+	}
+
+	formatter := &logrus.TextFormatter{
+		ForceColors:     true,
+		FullTimestamp:   true,
+		TimestampFormat: "2006-01-02 15:04:05.000000000",
```

Comment:
Our log's TimeStampFormat has been changed to "time.RFC3339Nano" and delete ForceColors, can you change this format like this:
```
	formatter := &logrus.TextFormatter{
		FullTimestamp:   true,
		TimestampFormat: time.RFC3339Nano,
	}
```